docs/source/libmav/libmav.py

```diff
@@ -0,0 +1,1054 @@
+        conn_runtime = libmav.NetworkRuntime(self.message_set, heartbeat, conn_physical)
+
+        # Return Connection or throw Exception
+        connection = conn_runtime.await_connection(2000)
+        ```
+
+        Args: 
+            timeout (int): Time to wait for connection before raising exception, in milliseconds
+
+        Returns:
+            Connection: An object representing a connection to the MAVLink network.
+        """
+        pass
+
+    def clear_heartbeat_message(self):
+        """Clear the heartbeat message (if set).
+        
+        This stops streaming the heartbeat message, which may have been set in the class initializer or using `set_heartbeat_message()`.
+        
+        ```python
+        connection.clear_heartbeat_message()
+        ```
+        """
+        pass
+        
+        
+    def on_connection(self, callbackFn):
+        """Set a callback function that is called whenever a new `Connection` is detected.
+        
+        A Connection is created if MAVLink traffic is detected from a unique remote address/port.
+        
+        The callback function is called with the `Connection` object.
+        The identify of the unique `ConnectionPartner` associated with the `Connection` be determined using the `Connection.partner()` method.
+        Note that a `NetworkRuntime` constructed using a `TCPClient`, `UDPClient` or `Serial` network interface will only have one connection (because there is only one partner), while a `TCPServer` or `UDPServer` might have several remote systems connecting from different IP addresses.
+        
+
+       ```python
+       # Define a callback function
+       def callbackConnected(connection):
+           print(f'New connection: {connection}')
+    
+       conn_runtime.on_connection(callbackConnected)        
+        ```
+        
+        Args: 
+            callbackFn (method): Callback function that takes a `Connection` object as its argument.
+    
+        """
+        pass
+
+
+    def on_connection_lost(self): # not sure of signature yet
+        """Set a callback function that is called whenever a `Connection` is lost.
+        
+        A connection is lost if no data is detected from corresponding connection partner within three seconds (i.e. it is no longer returning true for `Connection.alive()`).
+        
+        The callback function is called with the `Connection` object for the lost connection.
+        The identify of the `ConnectionPartner` can be determined using the `Connection.partner()` method.
+        
+
+       ```python
+       # Define a callback function
+       def callbackDisonnected(connection):
+           print(f'Disconnected: {connection}')
+    
+       conn_runtime.on_connection_lost(callbackDisonnected)        
+        ```
+        
+        
+        Args: 
+            callbackFn (method): Callback function that takes a `Connection` object as its argument.
+    
+        """
```

Comment:
```suggestion
        """Set a callback function that is called whenever a `Connection` is lost.
        
        A connection is lost if no decodable MAVLink messages are detected from the corresponding connection partner within three seconds (i.e. it is no longer returning `true` for `Connection.alive()`).
        
        The callback function is called with the `Connection` object for the lost connection.
        The identify of the `ConnectionPartner` can be determined using the `Connection.partner()` method.

        ```python
        # Define a callback function
        def callbackDisconnected(connection):
            print(f'Disconnected: {connection}')
    
        conn_runtime.on_connection_lost(callbackDisconnected)        
        ```

        Note that if you lose a connection and want to recover it, you will need to recreate the physical interface and the `NetworkRuntime`, and then wait for the `Connection` again.

        Args: 
            callbackFn (method): Callback function that takes a `Connection` object as its argument.
    
        """
```